### PR TITLE
Theme builder toaster prep

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

**Review by kobizz — DISMISSED** [review]

File: assets/dev/js/editor/document/save/hooks/ui/save/after.js [hunk 1]
```diff
@@ -28,24 +28,25 @@ export class FooterSaverAfterSave extends HookUIAfter {
 	}
 
 	onPageStatusChange( newStatus ) {
-		if ( 'publish' === newStatus ) {
-			const buttons = [];
+		// Don't add the "Have a look" link in the theme builder.
+		if ( elementor.config.document.support_site_editor || 'publish' !== newStatus || ! elementor.config.document.urls.have_a_look ) {
```
Comment (matipojo) [inline assets/dev/js/editor/document/save/hooks/ui/save/after.js:32]: Move to hook conditions.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/document/save/hooks/ui/save/after.js (hunk 1 above)
Comment (udidol) [reply assets/dev/js/editor/document/save/hooks/ui/save/after.js:32]: Are you sure? @matipojo 
These conditions only apply to the toaster, not the rest of the things in the `apply()` method

**Review by kobizz — DISMISSED** [review]

File: assets/dev/js/editor/document/save/hooks/ui/save/after.js [hunk 2]
```diff
@@ -28,24 +28,25 @@ export class FooterSaverAfterSave extends HookUIAfter {
 	}
 
 	onPageStatusChange( newStatus ) {
-		if ( 'publish' === newStatus ) {
-			const buttons = [];
+		// Don't add the "Have a look" link in the theme builder.
```
Comment (matipojo) [inline assets/dev/js/editor/document/save/hooks/ui/save/after.js:31]: Can you discard all changes?

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please merge to one commit.

**Pull request merged**